src/Licenses/Entities/License/Prediction.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Licenses\Entities\License;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+use ArrowSphere\PublicApiClient\Exception\EntityValidationException;
+
+/**
+ * Class ActiveSeats
+ */
+class Prediction extends AbstractEntity
+{
+    public const COLUMN_CURRENCY = 'currency';
+    public const COLUMN_UPDATE_DATE_OF_ML_SCRIPT = 'updatedAt';
+    public const COLUMN_LICENSE_REFERENCE = 'licenseRef';
```

Comment:
If I read the specs.yaml of your endpoint, this field is named "licenseReference"
```suggestion
    public const COLUMN_LICENSE_REFERENCE = 'licenseReference';
```